layouts/css/_variables.styl

```diff
@@ -2,9 +2,10 @@ $body-max-width = 980px
 
 $white = #fff
 
-$node-green = #80bd01
-$light-green = #f1fbda
-$hover-green = #d9ebb3
+$node-green = #43853d
```

Comment:
Well, It's barely noticeable, so I don't have much of an opinion, but I'd lean on more the side of using the original color than to make it match the guideline.